compiler-cli/src/cli.rs

```diff
@@ -185,7 +185,7 @@ pub fn stdout_buffer_writer() -> BufferWriter {
 }
 
 fn color_choice() -> ColorChoice {
-    if atty::is(atty::Stream::Stderr) {
+    if atty::is(atty::Stream::Stderr) || std::env::var("FORCE_COLOR").is_ok() {
```

Comment:
`std::env::var("FORCE_COLOR").is_ok(()` returns `true` when `FORCE_COLOR` is an empty string. It should be `false`.

> When this variable is present and **not an empty string** (regardless of its value), it should force the addition of ANSI color.
	https://force-color.org/

Reply:
Yes please!

Reply:
Done